client/landing/gutenboarding/stepper-onboarding/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { Route, Switch } from 'react-router-dom';
+import { useStepNavigation } from './hooks/use-step-navigation';
+import { Steps } from './steps';
+
+export function Stepper() {
+	const { goNext, goBack } = useStepNavigation();
+
+	return (
+		<div>
+			{ Object.values( Steps ).map( ( step ) => (
+				<Switch>
+					<Route path={ `/${ step }` } key={ step }>
+						{ step } step
+					</Route>
+				</Switch>
+			) ) }
+			<div>
+				<button onClick={ goBack }>Previous step</button>
+				<button onClick={ goNext }>Next step</button>
```

Comment:
I'd recommend throwing in some random translated text, just so we can confirm that the l10n machinery is set up (it's not, see comment about the `403` error).

```suggestion
				<button onClick={ goBack }>{ __( 'Previous' ) }</button>
				<button onClick={ goNext }>{ __( 'Next' ) }</button>
```